Makefile

```diff
@@ -11,12 +11,20 @@ GORUN = env GO111MODULE=on go run
 libzkp:
 	cd $(PWD)/rollup/circuitcapacitychecker/libzkp && make libzkp
 
+libzstd:
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+	@sudo mv libzktrie.so /usr/local/lib
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
+	@sudo mv libscroll_zstd.so /usr/local/lib
```

Comment:
i will not recommend `mv XXX  /usr/local/lib`, it will overwrite `libzktrie.so` and `libscroll_zstd.so` if other people are using it

Reply:
it makes sense not to override `libzktrie.so`, because the version could be different.
moving to an un-default dir needs to assign an explicit path in `LDFLAGS` in da-codec. any common path to set this value?

`LDFLAGS` here: https://github.com/scroll-tech/da-codec/blob/c4d0a7038d49bc07fcf560379d531a7e400d2152/encoding/codecv2/codecv2.go#L4

Reply:
removed.